utils/templates.py

```diff
@@ -261,3 +261,55 @@ def __init__(self, params=None, content=None):
 
         self.body = copy.deepcopy(self.params)
         self.body.update({'content': self.content})
+
+class ShortReviewReply(InvitationReply):
+    """
```

Comment:
I think the longer term solution is actually to change the invitation options in the backend, to allow us to create a single invitation for reviews that is attached to all invitations of a certain ID. In the most basic conference, this will remove the need for the "create_invitations" script entirely.

it still might be a good idea to add a "create_invitations" template to the script generator, but it may not be necessary for the most basic conference configuration